src/cursor_manager.ts

```diff
@@ -177,7 +181,11 @@ export class CursorManager implements Disposable {
      * Called when cursor update received. Waits for document changes to complete and then updates cursor position in editor.
      */
     private processCursorMoved(): void {
-        for (const gridId of this.gridCursorUpdates) {
+        for (const [gridId, shouldUpdate] of this.gridCursorUpdates.entries()) {
+            if (!shouldUpdate) {
```

Comment:
```suggestion
            if (!shouldUpdate()) {
```

Reply:
Maybe I'll open another PR to add some type-checked rules to eslint like https://typescript-eslint.io/rules/no-unnecessary-condition/